metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/BatchIngestionRunResource.java

```diff
@@ -221,6 +241,30 @@ private void sleep(Integer seconds) {
     }
   }
 
+  private void updateExecutionRequestStatus(String runId, String status) {
+    try {
+      final Urn executionRequestUrn = EntityKeyUtils.convertEntityKeyToUrn(new ExecutionRequestKey().setId(runId), Constants.EXECUTION_REQUEST_ENTITY_NAME);
+      EnvelopedAspect aspect =
+          _entityService.getLatestEnvelopedAspect(executionRequestUrn.getEntityType(), executionRequestUrn, Constants.EXECUTION_REQUEST_RESULT_ASPECT_NAME);
+      if (aspect == null) {
+        log.warn("Aspect for execution request with runId {} not found", runId);
+      } else {
+        final MetadataChangeProposal proposal = new MetadataChangeProposal();
+        ExecutionRequestResult requestResult = new ExecutionRequestResult(aspect.getValue().data());
+        requestResult.setStatus(status);
+        proposal.setEntityUrn(executionRequestUrn);
+        proposal.setEntityType(Constants.EXECUTION_REQUEST_ENTITY_NAME);
+        proposal.setAspectName(Constants.EXECUTION_REQUEST_RESULT_ASPECT_NAME);
+        proposal.setAspect(GenericRecordUtils.serializeAspect(requestResult));
+        proposal.setChangeType(ChangeType.UPSERT);
+
+        _entityService.ingestProposal(proposal, new AuditStamp().setActor(UrnUtils.getUrn(Constants.SYSTEM_ACTOR)).setTime(System.currentTimeMillis()));
+      }
+    } catch (Exception e) {
+      log.warn(String.format("Not able to update execution result aspect with runId %s.", runId), e);
```

Comment:
let's do log.error here! 

let's also log the status that failed to update so we know how to address this